tools/fw-update/rs-fw-update.cpp

```diff
@@ -119,9 +120,23 @@ void update(rs2::update_device fwu_dev, std::vector<uint8_t> fw_image)
     std::cout << std::endl << std::endl << "Firmware update done" << std::endl;
 }
 
-void list_devices(rs2::context ctx)
+void list_devices( rs2::context ctx, bool only_sw_devs )
 {
     auto devs = ctx.query_devices();
+    if( only_sw_devs )
+    {
+        // For SW-only devices, allow some time for DDS devices to connect
+        int tries = 3;
+        std::cout << "No device detected. Waiting..." << std::flush;
```

Comment:
Some devices can already be detected. "Waiting for remote devices" might be better

Reply:
OK, will change.